packages/patternfly-4/react-core/src/components/List/List.tsx

```diff
@@ -2,26 +2,40 @@ import * as React from 'react';
 import styles from '@patternfly/react-styles/css/components/List/list';
 import { css, getModifier } from '@patternfly/react-styles';
 
+export type OrderTypes = '1' | 'a' | 'A' | 'i' | 'I';
+
 export enum ListVariant {
-  inline = 'inline'
+  inline = 'inline',
+  ordered = 'ordered',
+  unordered = 'unordered',
 }
 
-export interface ListProps extends React.HTMLProps<HTMLUListElement> {
+export interface ListProps extends React.HTMLProps<HTMLOListElement | HTMLOListElement> {
```

Comment:
`<HTMLOListElement | HTMLOListElement>` typo ?
should be:
`<HTMLUListElement | HTMLOListElement>`